x-pack/plugins/rule_registry/server/authorization/audit_logger.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AuditLogger, EventCategory, EventOutcome, EventType } from '../../../security/server';
+
+export enum AuthorizationResult {
+  Unauthorized = 'Unauthorized',
+  Authorized = 'Authorized',
+}
+
+export class RacAuthorizationAuditLogger {
+  private readonly auditLogger: AuditLogger;
+
+  constructor(auditLogger: AuditLogger = { log() {} }) {
+    this.auditLogger = auditLogger;
+  }
+
+  public getAuthorizationMessage(
+    authorizationResult: AuthorizationResult,
+    owner: string,
+    operation: string
+  ): string {
+    return `${authorizationResult} to ${operation} "${owner}" alert"`;
```

Comment:
Is this meant to be shown to users, including perhaps API users? If so, should this be internationalized?

Reply:
Yes server-side internationalization should definitely be happening. I haven't seen examples of this before though. If anyone has any to point to that'd be helpful but for sure would love to have this.

Reply:
I think, historically, we haven't applies i18n to Server and Audit logs.... 🤔 
I recall that there's a reason, but I can't recall what it is 🤦 

(poking old timers with tribal knowledge @kobelb @legrego )

Reply:
The weaknesses of tribal knowledge are apparent here because I too recall us not i18n'ing error messages or audit log entries; however, I don't recall the reasons.

Reply:
I'm guessing from a supportability / SDH perspective; it would be hard to understand and help based on the server logs?

Reply:
This code is concatenating strings as it is, so I don't think doing that using an i18n library would make it any harder to read. If the message is intended for CLI logs and the such, then I agree that they don't need to be i18n'd. But if the message is being returned in an HTTP response then I think we should consider i18n'ing it.

Reply:
@dhurley14 @gmmorris @mikecote I opened a discuss issue to talk about this: https://github.com/elastic/kibana/issues/97650